exp/controllers/awsmanagedmachinepool_controller.go

```diff
@@ -139,11 +140,25 @@ func (r *AWSManagedMachinePoolReconciler) Reconcile(req ctrl.Request) (_ ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
+	awsCluster := &infrav1.AWSCluster{}
```

Comment:
We don't have a `AWSCluster` for the managed machinepool and we are also trying to get rid of `AWSManagedCluster` as well

Reply:
We should chat about this one